plugins/catalog-backend/src/service/router.ts

```diff
@@ -84,6 +84,11 @@ export async function createRouter(
         const output = await locationsCatalog.locations();
         res.status(200).send(output);
       })
+      .get('/locations/:id/history', async (req, res) => {
+        const { id } = req.params;
+        const output = await locationsCatalog.locationHistory(id);
+        res.status(200).send(output);
```

Comment:
no need to set status here imo, 200 is the default

Reply:
Explicit is better than implicit?)